fluss-common/src/main/java/com/alibaba/fluss/metadata/TableInfo.java

```diff
@@ -149,6 +149,11 @@ public boolean hasPrimaryKey() {
         return !primaryKeys.isEmpty();
     }
 
+    /** Check if the table is a primary key table or a log table. */
+    public boolean isPrimaryKeyTable() {
```

Comment:
if `hasPrimaryKey` return true, it means it's a primaryKeyTable. We don't need to introduce this method.